[debug] [2020-04-03T18:25:59.923Z] ----------------------------------------------------------------------
[debug] [2020-04-03T18:25:59.926Z] Command:       /usr/local/bin/node /usr/local/bin/firebase deploy --only hosting:onlinestore-grp
[debug] [2020-04-03T18:25:59.927Z] CLI Version:   8.0.0
[debug] [2020-04-03T18:25:59.927Z] Platform:      darwin
[debug] [2020-04-03T18:25:59.927Z] Node Version:  v10.16.3
[debug] [2020-04-03T18:25:59.928Z] Time:          Fri Apr 03 2020 13:25:59 GMT-0500 (GMT-05:00)
[debug] [2020-04-03T18:25:59.928Z] ----------------------------------------------------------------------
[debug] [2020-04-03T18:25:59.928Z] 
[debug] [2020-04-03T18:25:59.957Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-03T18:25:59.957Z] > authorizing via signed-in user
[debug] [2020-04-03T18:25:59.958Z] [iam] checking project onlinestore-3fc65 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2020-04-03T18:25:59.960Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2020-04-03T18:25:59.961Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2020-04-03T18:26:01.347Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Fri, 03 Apr 2020 18:26:02 GMT","server":"scaffolding on HTTPServer2","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"quic=\":443\"; ma=2592000; v=\"46,43\",h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-04-03T18:26:01.364Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/onlinestore-3fc65:testIamPermissions  
 {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2020-04-03T18:26:02.003Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Fri, 03 Apr 2020 18:26:03 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=468","alt-svc":"quic=\":443\"; ma=2592000; v=\"46,43\",h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-04-03T18:26:02.005Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/onlinestore-3fc65  
 
[debug] [2020-04-03T18:26:02.291Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Fri, 03 Apr 2020 18:26:03 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"quic=\":443\"; ma=2592000; v=\"46,43\",h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000","accept-ranges":"none","transfer-encoding":"chunked"}
[info] 
[info] === Deploying to 'onlinestore-3fc65'...
[info] 
[info] i  deploying hosting
[info] 
[info] âœ”  Deploy complete!
[info] 
[info] Project Console: https://console.firebase.google.com/project/onlinestore-3fc65/overview
[debug] [2020-04-03T18:26:02.489Z] TypeError: Cannot read property 'deploys' of undefined
    at /usr/local/lib/node_modules/firebase-tools/lib/deploy/index.js:84:36
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
